deps/unwind.mk

```diff
@@ -30,10 +30,6 @@ endif
 
 define LIBUNWIND_INSTALL
```

Comment:
can you make that commit here? likewise with the clean-fftw issue mentioned separately, unless you spell out your intentions in full detail it's not easy to see exactly what you mean